src/sonic-yang-models/yang-models/sonic-pbh.yang

```diff
@@ -25,7 +29,7 @@ module sonic-pbh {
 
 	typedef hash-field {
 		description "Represents native hash field";
-		type enumeration {
+		type stypes:hash-field {
 			enum INNER_IP_PROTOCOL;
```

Comment:
Still using enum?

Reply:
@ganglyu correct .A subset is used to enable only fields which are relevant for this feature.